docs/zero_to_hero_guide/quickstart.md

```diff
@@ -0,0 +1,185 @@
+# Llama Stack Quickstart Guide
+
+This guide will walk you through setting up an end-to-end workflow with Llama Stack, enabling you to perform text generation using the `Llama3.2-3B-Instruct` model. Follow these steps to get started quickly.
+
+If you're looking for more specific topics like tool calling or agent setup, we have a [Zero to Hero Guide](#next-steps) that covers everything from Tool Calling to Agents in detail. Feel free to skip to the end to explore the advanced topics you're interested in.
+
+## Table of Contents
+1. [Setting up](#Setting-up)
+2. [Build, Configure, and Run Llama Stack](#build-configure-and-run-llama-stack)
+3. [Testing with `curl`](#testing-with-curl)
+4. [Testing with Python](#testing-with-python)
+5. [Next Steps](#next-steps)
+
+---
+
+
+
+## Setting up
+
+### 1. Prerequisite
```

Comment:
I don't think this should have 1. 2. 3. inside at all. Just one `Setup` section and perhaps a small subsection on downloading models.

Reply:
hi @ashwinb, same as previous comment ask well I’m happy to remove the subsections, but I believe numbering makes the steps easier to follow